core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -1710,36 +1710,74 @@ check_linked_symbol(WASMModuleInstance *module_inst, char *error_buf,
             && !func->import_func_linked
 #endif
         ) {
-#if WASM_ENABLE_WAMR_COMPILER == 0
             LOG_WARNING("warning: failed to link import function (%s, %s)",
                         func->module_name, func->field_name);
-            /* will throw exception only if calling */
-#else
-            /* do nothing to avoid confused message */
-#endif /* WASM_ENABLE_WAMR_COMPILER == 0 */
         }
     }
 
     for (i = 0; i < module->import_global_count; i++) {
         WASMGlobalImport *global = &((module->import_globals + i)->u.global);
-        if (!global->is_linked) {
+
+        if (!global->is_linked
+            && !wasm_runtime_is_built_in_module(global->module_name)) {
```

Comment:
wondering why need to check `wasm_runtime_is_built_in_module()` here. IIUC, if requires a built_in module, it should be satisfied in *wasm_loader.c*.

Reply:
I am not sure, I just refactored the code based on @XeniaLu's original modification. Maybe @XeniaLu can help remove this line to have a try?

Reply:
Sure, I will remove it.